proposals/0240-sdl-js-pwa.md

```diff
@@ -345,12 +345,12 @@ Many services are available over a web application and modern WebEngines provide
   </param>
 </function>
 </interface>
-```
+```f
 
 ### Example manifest file
 
 ```json
-{
+export const sdl_manifest = {
```

Comment:
Should likely be something like `export { sdl_manifest };` or `export default { sdl_manifest };`. Suggest noting this to be an implementation detail.